### DISCUSSION TITLE
Tidepool no longer receiving bolus and carb info

### DISCUSSION THREAD

**Discussion by jgslade** [NightscoutFoundation/xDrip#2628] [General]
When trying to prepare for my endo visit when I looked at tidepool the Bolus & Carbohydrates section was empty, BGs are present and Basal Rates are present, but the Bolus and Carbs are not there, in reviewing historical data it looks like they stopped being uploaded on 14 Sep 2022. I don't look at my historical data often just when prepping for endo visits. looking at my apks for AAPS I didn't have a new build on the 13th or 14th, so I don't think it's an issue there.

**Comment by jgslade** [comment]
Looking at the history in Xdrip it looks like Xdrip doesn't have the Bolus and Carbs after 13 Sep 2022. So it's not an issue of the uploader, but the importer...

**Comment by simonpickering** [comment]
It may be related to this: #2597

**Comment by jgslade** [comment]
I think the issue was because AAPS was having issues uploading to my nightscout site, so wasn't sending the bolus and carbs to xdrip.  Once I got my nightscout uploads working properly xdrip started getting the bolus and carb info.

One issue is still remaining in that it doesn't look like smbs are being recorded or sent to tidepool. Though I think there is a discussion about this problem, so I'll look into that.

ETA: Looked and it looks like there isn't anything about smbs not being recorded in tidepool.

**Reply by kleenkanteen** [reply]
what is smbs

**Reply by jgslade** [reply]
Super micro boluses, probably should have capitalized it, SMBs.
